README.md

```diff
@@ -46,6 +46,7 @@ Sizes of the `vlitejs` bundle compared to the competition:
 - [**Cast**](./src/plugins/cast/README.md) - Supports for Google Cast API.
 - [**AirPlay**](./src/plugins/airplay/README.md) - Supports for Apple AirPlay API.
 - [**Monetization**](./src/plugins/ima/README.md) - Supports for Google IMA SDK.
+- [**Mobile**](./src/plugins/mobile/README.md) - Supports for mobile.
```

Comment:
Not sure for the plugin name. Can be confusing with the `hotkey` plugin?

Perhaps `fastforward`?